schemas/base.yml

```diff
@@ -14,8 +14,11 @@
       description: >
         Date/time when the event originated.
 
-        For log events this is the date/time when the event was generated, and
-        not when it was read.
+        This is the date/time extracted from the event, representing when the
```

Comment:
should probably say "This is the date/time extracted from the event, typically representing when the"